File: app/models/day.rb
class Day
  include Mongoid::Document
  
  referenced_in :card, :inverse_of => :days
  
  field :date, :type => Date
  field :notes, :default => ""
  field :worked, :default => "Worked"
  
  validates_uniqueness_of :date
  
end                                    

File: time_card/apps/time_card/models/day.js
// ==========================================================================
// Project:   TimeCard.Day
// Copyright: ©2010 My Company, Inc.
// ==========================================================================
/*globals TimeCard */

/** @class

  (Document your Model here)

  @extends SC.Record
  @version 0.1
*/
TimeCard.Day = SC.ChildRecord.extend(
/** @scope TimeCard.Day.prototype */ { 
  
  type: 'Day',

  worked: SC.Record.attr(String),
  date: SC.Record.attr(Date),
  note: SC.Record.attr(String),
  _type: SC.Record.attr(String),
  _id: SC.Record.attr(String),
  card: SC.Record.toOne("TimeCard.Card", {
    inverse:'days', isMaster: NO
  })

}) ;
